plan/service.py

```diff
@@ -94,7 +94,7 @@ def monthly_uploads_limit(self) -> Optional[MonthlyUploadLimits]:
         return self.plan_data.monthly_uploads_limit
 
     # Trial Data
-    def start_trial(self) -> None:
+    def start_trial(self, current_owner) -> None:
```

Comment:
Followup to Jerry's comment, could you type this current_owner var pls

Reply:
hmm the use of current_owner is a core concept of the API now, so my thinking was if we use current_owner it would be explicit we're referring to the owner model, will update!

Reply:
Yeaah, it's sort of core to us I suppose, typing your variables in general is a good idea to keep readability and help maneuver through the IDE! It'd be basically 

`def start_trial(self, current_owner: Owner) -> None:`